### DISCUSSION TITLE
Displaying university logo

### DISCUSSION THREAD

**Discussion by XtayCalmEco** [nmfs-opensci/quarto-thesis#18] [General]
I input a file location and yet nothing appears to show - do you have any advice?

**Comment by jacintomathe** [comment]
I am facing the same problem. If you have any advice would be more than appreciated.

**Comment by jacintomathe** [comment]
I also wonder how I can adjust the font size of the chapters; they appear extremely large.  Thank you for this.

**Reply by eeholmes** [reply]
Hi, Can you give an example of a repository or the yaml so I can see the problem? It cannot find the file but I need to see exactly what you are doing to help.

**Reply by jacintomathe** [reply]
Hi! Thank you for getting back. 

Below is the same repository I am trying to customise for my Thesis. How would you add university logo for instance the image in the images folder?

And how can I set the font size of the chapters? 

[Mathe_thesis.zip](https://github.com/nmfs-opensci/quarto-thesis/files/13679084/Mathe_thesis.zip)

**Comment by eeholmes** [comment]
@jacintomathe there was a bug in the extension.

update the extension with
```
quarto install extension nmfs-opensci/quarto-thesis
```
The logo is now in the `thesis` section of `_quarto.yml`
